### ISSUE TITLE
Is it possible to save the "mapping"?

### ISSUE THREAD

**Issue by romaen** [saqneo/Elite#4]
Hello! 
First of all, thanks for the tool, it's awesome!
My question, is it possible to save the "mapping"? I don't want to set the keys everytime I restart the tool/pc.
Thank u!

**Comment by saqneo** [comment]
Unfortunately, there is currently no way to do this. Out of curiosity, do
you have this working on the Anniversary Update? A lot of people were
having trouble, but I may add a few more features - such as saving - if it
is working for some.

**Comment by romaen** [comment]
No, I dont have the Anniversary Update installed, but still - I had troubles to "install" the app. I had the same error like a lot of people:
 "EliteUi_1.0.0.0_x64.appx and Add-AppDevPackage.ps1 not in expected build location C:\users\XXX\desktop\elite-master\elite\EliteUi\Out\EliteUi\AppPackages\EliteUi_1.0.0.0_x64_Test.............."

I fixed that with deinstalling all old SDKs and only installing the one, which VisualStudio recommended after starting "Elite.sln" (with VisualStudio). Maybe that help other ppl too.
